index.js

```diff
@@ -45,12 +45,24 @@ const submitAudit = async opts => {
   }
 };
 
+const submitDuplicate = async opts => {
+  try {
+    if (config.audit) {
+      await db('duplicates').insert(opts);
+    }
+  } catch (e) {
+    const id = opts.case_id || 'N/A (Failed to send)';
+    logError(`iCasework Case ID ${id}`, 'Duplicate', e);
+    throw new Error('Duplicate Table Error');
```

Comment:
a lot of duplication here. Logging the duplicate log is still for audit purposes. So we should reuse the `submitAudit` method and pass maybe another argument to it.

Reply:
Add a 'type' parameter to the submitAudit method and do
```
await db(type).insert(opts);
```

Reply:
o yes we were meant to refactor this, forgot, thanks for spotting this one 🙂